featuretools/utils/wrangle.py

```diff
@@ -100,20 +100,6 @@ def _check_time_against_column(time, time_column):
     return False
 
 
-# --> TODO possibly remove this or convert to Woodwork
-def _check_time_type(time):
-    '''
-    Checks if `time` is an instance of common int, float, or datetime types.
-    Returns "numeric", "datetime", or "unknown" based on results
-    '''
-    time_type = None
-    if isinstance(time, (datetime, np.datetime64)):
-        time_type = "datetime_time_index"
-    elif isinstance(time, (int, float)) or np.issubdtype(time, np.integer) or np.issubdtype(time, np.floating):
-        time_type = "numeric_time_index"
-    return time_type
```

Comment:
Do we also want to remove `_check_cutoff_time_type` since it uses `_check_time_type`?
https://github.com/alteryx/featuretools/blob/dc5f69df4ad9a52aa0de581aa741a415bb32aaf1/featuretools/computational_backends/utils.py#L251-L273

Reply:
@jeff-hernandez I think we will still need `_check_cutoff_time_type` to make sure that the cutoff time type is  consistent with the time index type specified in the Woodwork dataframes. Maybe the better approach here is to keep `_check_time_type` and `_check_cutoff_time_type` and refactor to use the updated values that we will have for `es_time_type` and to also work if the cutoff time is a WW dataframe (in the block that has a `NotImplementedError()` now).

What do you think?

Reply:
+1 on keeping both functions and refactoring to use `'numeric'` and `Datetime`! (didn't realize that _check_time_type was used in this way, I think, when I was originally thinking we could remove it)

Reply:
Sounds good, updated the values in `_check_time_type`